test/control-plane.test.ts

```diff
@@ -90,69 +86,3 @@ describe('ControlPlane Targets', () => {
     } while (targetsCapture.next());
   });
 });
-
```

Comment:
Were these tests not helpful?

Reply:
added tests back, but checks for access log settings instead of role.